lib/spack/spack/cmd/create.py

```diff
@@ -41,16 +38,37 @@
 from spack.spec import Spec
 from spack.util.naming import *
 from spack.repository import Repo, RepoError
-import spack.util.crypto as crypto
 
 from spack.util.executable import which
-from spack.stage import Stage
 
 
 description = "Create a new package file from an archive URL"
 
-package_template = string.Template(
-    _copyright + """
+package_template = string.Template("""\
```

Comment:
why duplicate the copyright statement?  This is what i put in `_copyright` for -- so you could just get it from the top of the file.